AppCenter/AppCenter/AppCenter.modulemap

```diff
@@ -0,0 +1,6 @@
+framework module AppCenter {
```

Comment:
Can we reuse existing one? I'd keep only one modulemap to avoid confusions

Reply:
Those were not necessary at all - since we provided include symlinks.